Library/Formula/icecast.rb

```diff
@@ -1,5 +1,3 @@
-require 'formula'
-
```

Comment:
To make merging with Homebrew easier, patches to Linuxbrew should not include unnecessary changes. Changes such as these can be submitted upstream to Homebrew, but they do not accept style-only changes.